play/src/lib/HelperFunctions.tsx

```diff
@@ -129,4 +130,33 @@ export const fetchLocalData = () => {
   if (!localModel)
     window.localStorage.removeItem(StorageKey);
   return localModel;
+};
+
+/**
+ * sorts teams by score descending, then alphabetically by name
+ * only include teams with scores in the top five
+ * @param inputTeams - the teams to be sorted
+ * @returns - the sorted teams
+ */ 
+export const teamSorter = (inputTeams: ITeam[]) => {
```

Comment:
This can be added as an extension to Array type in typescript, [this SO](https://stackoverflow.com/a/12803141/1450348) has some info, that'd be cleaner as then you can use this method by `inputTeams.sortByScore(5)` and also have the total number to be defined by the caller, like if in future you wanted to use this method, but only for top 2 teams, then you can

Reply:
I might save the extension to the Array type for a future PR but have added a param for the number of teams to display so that it can be tuned per your suggestion